config/routes.rb

```diff
@@ -338,12 +330,10 @@
         sort_vc_grid
         sort_vm_grid
         st_catalog_edit
-        st_catalog_form_field_changed
         st_edit
         st_form_field_changed
         st_tags_edit
         st_upload_image
-        tag_edit_form_field_changed
```

Comment:
This is being used in `app/views/layouts/_tag_edit.html.haml`, can you also remove it from there?

Reply:
done 👍